ActindoWorkFlow/Operation/05_UpdateDeployedVersion.md

```diff
@@ -0,0 +1,58 @@
+# Update a deployed version
+
+The *Update deployed version* function is designed to support in the test phase. Updating a deployed version does not create a new version, but overwrites the current version. This means that changes to the workflow are not recorded. For example, this function is suitable, if one of the included transitions is running on an error because of faulty configuration, and you do not want to set up the whole workflow again.       
```

Comment:
to support in the test phase? --> Ich verstehe nicht genau, was damit gemeint ist... Ich vermute, es hilft bei der Konfiguration/dem Design des Workflows, da man Änderungen "on the fly" vornehmen kann, ohne das Workflow neu erstellen zu müssen. Vielleicht so was in der Richtung? This function makes it easier for you to design and test your workflow...? 
this function is suitable, if... --> Da würde ich kein Komma setzen.